azerothcore/azerothcore-wotlk
# fix(DB): Various database cleanups

With this PR I want to fix some DB bugs that I have tracked in the game. 

Removing "[DEPRECATED]" quest titles
--------------------------
Some quests which are live ingame have a title like "[DEPRECATED]" or "Deprecated:". 
Many of these quests are active and can be completed.  
I just removed the strings that contain "deprecated" and updated the entries.  

Removing Highvale Ranger titles "PH MODEL WIP"
------------------------------------------------------------

Those NPCs are found in Hinterlands .go 324.5412129 -2677.939453 117.246620 0 and now have this title:

![image](https://user-images.githubusercontent.com/55922592/71196024-6cd83d80-228f-11ea-9c4d-a5336b0485d3.png)

This title have to be removed as the modoel is already up to date! 

**Source:** https://wow.gamepedia.com/Highvale_Ranger (See former appearance)

The title was also removed on Rising Gods P-Server (Best WotLK server for german language over 10 years now) and is declared as db error:

https://redmine.rising-gods.de/issues/22936



Old Model:
![image](https://user-images.githubusercontent.com/55922592/71195751-defc5280-228e-11ea-8d62-03ab598fef86.png)

Reworked (actual) Model:
![image](https://user-images.githubusercontent.com/55922592/71195789-efacc880-228e-11ea-8b1e-c5b424265bcb.png)






##### CHANGES PROPOSED:
- removed `Title` LIKE '%deprecated%' in quest_template_locale for deDE which are NOT included in disables table  - See the comments in File: Many of them were completed by myself ingame (34)
- removed `Title` LIKE '%WIP%' in creature_template_locales for all locales (8)

-  Fixed the weapon in offhand



##### ISSUES ADDRESSED:
- Closes


##### TESTS PERFORMED:
- Imported on win 10 pro 64 & HeidiSQL 10.3.0.5771 without errors and checked the changes



##### HOW TO TEST THE CHANGES:


- Just import the sql file with an GUI of your trust and check that no error occurs.

##### KNOWN ISSUES AND TODO LIST:


##### Target branch(es):




 

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
